java-checks/src/main/java/org/sonar/java/checks/NullPointerCheck.java

```diff
@@ -91,7 +91,7 @@ public void scanFile(final JavaFileScannerContext context) {
     this.context = context;
     semanticModel = (SemanticModel) context.getSemanticModel();
     if (semanticModel != null) {
-      context.getTree().accept(this);
+//      context.getTree().accept(this);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)